src/modules/m_filter.cpp

```diff
@@ -104,8 +106,11 @@ class FilterResult
 				case 'c':
 					flag_strip_color = true;
 				break;
+				case 'r':
+					flag_no_registered = true;
+				break;
 				case '*':
-					flag_no_opers = flag_part_message = flag_quit_message =
+					flag_no_opers = flag_no_registered = flag_part_message = flag_quit_message =
```

Comment:
This is probably too wide to be included with the default flags. Can you remove it for now we can always add it back later if necessary.

Reply:
I've also modified helpop.conf.example and inspircd.conf.example, i hope can be good my english is so bad ...